lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -433,7 +433,7 @@
             <?php if ($register_voters) : ?>
               <p><?php print $voter_reg_share_modal_text_2 ?></p>
             <?php endif; ?>
-            <?php if ($show_social_share_link) : ?>
+            <?php if ($show_social_share_link == 1) : ?>
```

Comment:
@katiecrane I think the idea here that @sbsmith86 mentioned is that it is more readable as an optional feature instead of it just not existing.